src/gui/mzroll/eicwidget.h

```diff
@@ -192,7 +187,10 @@ public Q_SLOTS:
 	void peakMarkedEicWidget();
     void groupSet(shared_ptr<PeakGroup>);
     void compoundSet(Compound*);
-
+	void optionTicChecked(bool);
+	void optionBarPlotChecked(bool);
+	void optionBoxPlotChecked(bool);
```

Comment:
Please use 4 spaces instead of tabs. You can easily configure your code editor to automatically do this whenever you press the "Tab" key. Using spaces ensures that the code will look the same on everyone's editors (as well as on GitHub).

Reply:
done can you please check again?

Reply:
All good now.